vbeam/data_importers/pyuff_importer.py

```diff
@@ -100,12 +100,20 @@ def import_pyuff(
     if numpy.abs(modulation_frequency) == 0:
         receiver_signals = np.array(hilbert(receiver_signals), dtype="complex64")
 
-    receivers = ElementGeometry(
-        np.array(channel_data.probe.xyz),
-        np.array(channel_data.probe.theta),
-        np.array(channel_data.probe.phi),
-    )
-    sender = ElementGeometry(np.array([0.0, 0.0, 0.0], dtype="float32"), 0.0, 0.0)
+    if channel_data.probe.__class__ == 'pyuff_ustb.objects.probes.curvilinear_array.CurvilinearArray':
+        ROC_azimuth = channel_data.probe.radius
+    elif channel_data.probe.__class__ == 'pyuff_ustb.objects.probes.curvilinear_matrix_array.CurvilinearMatrixArray':
+        ROC_azimuth = channel_data.probe.radius_x
+    else:
+        ROC_azimuth = 10
+    ROC_elevation = 10
```

Comment:
I think we should either create a convention that if the ROC is `None`, then that means it is a flat probe, or create separate classes for linear and curvilinear probes. I am leaning more towards the latter.